content/ko/docs/reference/labels-annotations-taints.md

```diff
@@ -262,11 +263,29 @@ kube-controller-manager의 잡(Job) 컨트롤러는
 
 ## endpoints.kubernetes.io/over-capacity
 
-예시: `endpoints.kubernetes.io/over-capacity:warning`
+예시: `endpoints.kubernetes.io/over-capacity:truncated`
 
 적용 대상: 엔드포인트(Endpoints)
 
-v1.21 이상의 쿠버네티스 클러스터에서, 엔드포인트(Endpoints) 컨트롤러가 1000개 이상의 엔드포인트를 관리하고 있다면 각 엔드포인트 리소스에 이 어노테이션을 추가한다. 이 어노테이션은 엔드포인트 리소스가 용량 초과 되었음을 나타낸다.
+v1.22 이상의 쿠버네티스 클러스터에서, 엔드포인트(Endpoints) 컨트롤러가 1000개 이상의 엔드포인트를 관리하고 있다면 각 엔드포인트 리소스에 이 어노테이션을 추가한다. 이 어노테이션은 엔드포인트 리소스가 용량 초과 되었음을 나타낸다.
+
+## batch.kubernetes.io/job-tracking
+
+예시: `batch.kubernetes.io/job-tracking: ""`
+
+적용 대상: 잡
+
+잡에 어노테이션이 있으면 컨트롤 플레인은 [finalizers를 사용하여 잡 상태 추적](/ko/docs/concepts/workloads/controllers/job/#job-tracking-with-finalizers) 
+중임을 나타낸다.
+어노테이션을 수동으로 추가하거나 제거하지 **않는다**.
+
+## scheduler.alpha.kubernetes.io/preferAvoidPods (deprecated) {#scheduleralphakubernetesio-preferavoidpods}
+
+적용 대상: 노드
+
+이 어노테이션을 사용할려면 [NodePreferAvoidPods 스케줄링 플러그인](/ko/docs/reference/scheduling/config/#scheduling-plugins)이 활성화되어 있어야 한다.
+플러그인은 쿠버네티스 1.22 이후로 더 이상 사용하지 않는다.
```

Comment:
[원문]
> This annotation requires the NodePreferAvoidPods scheduling plugin to be enabled. The plugin is deprecated since Kubernetes 1.22.

```suggestion
해당 플러그인은 쿠버네티스 1.22에서 사용 중단되었다.
```